test/ScanDependencies/optional_transitive_dep_load_fail.swift

```diff
@@ -0,0 +1,49 @@
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/clang-module-cache)
+// RUN: %empty-directory(%t/Foo.swiftmodule)
+// RUN: %empty-directory(%t/inputs)
+// RUN: echo "@_implementationOnly import A; public func foo() {}" > %t/Foo.swift
+// REQUIRES: executable_test
+// REQUIRES: objc_interop
+
+// Create the explicit module inputs map
+// RUN: echo "[{" > %/t/inputs/map.json
```

Comment:
Probably not worth the trouble to change now, but it feels like this test would be a bit more maintainable if it leveraged `%split-file`.